traitlets/traitlets.py

```diff
@@ -311,15 +311,25 @@ class that does the following:
     name = None
     this_class = None
 
+    def top_instance_init(self, obj, **kwargs):
```

Comment:
I'd like to avoid adding yet another method, especially in a PR called 'Simplify...'